content/projects/java-specific/hello-sql-and-java/_index.md

```diff
@@ -1,3 +1,54 @@
 ---
-title: SQL + Java Helloworld
+title: SQL Extended
 ---
+
+We have had many sessions about JDBC Templates, JPA and Hibernate, so I would hope that you now have a clue of what we are talking about.
+
+Remember the scripts you submitted for this https://umuzi-org.github.io/tech-department/projects/sql/ project now we are going to do this through JAVA.
```

Comment:
rather use a `{{% contentlink` here. Chances are that the base url will change. so `https://umuzi-org.github.io/tech-department/projects/sql/` might break